packages/framework/store/src/store/meta.ts

```diff
@@ -162,6 +162,12 @@ export class DocCollectionMeta {
     });
   };
 
+  initialize() {
```

Comment:
might add call this in create doc is explicitly called in blocksuite?

Reply:
Done

Reply:
Roll back after discuss with @EYHN